package.json

```diff
@@ -27,5 +27,8 @@
   ],
   "dependencies": {
     "semver": "^7.5.0"
+  },
+  "msw": {
+    "workerDirectory": "example/react-mui-demo"
```

Comment:
does this need to be here if it is already in `/example/react-mui-demo/package.json`?